### DISCUSSION TITLE
Facing issue while installing node_modules

### DISCUSSION THREAD

**Discussion by Yuva0** [adobe/react-spectrum#6491] [Q&A]
Hello,

I am fairly new to this repository and wanted to run a build in my local but I am not able to install dependencies via yarn.
I am using Node v18.20.3 and running in the main branch.

This is the issue that I am getting.
<img width="1214" alt="image" src="https://github.com/adobe/react-spectrum/assets/33837714/c4585d99-c873-4d14-a0f4-1ccbf948fe8a">

Could someone help me resolve it?

**Comment by snowystinger** [comment]
Hmm I'm running node 18.20.3 and it's working fine for me
All you did was `git clone` followed by a `yarn`?
are you using yarn 1.22?

**Reply by Yuva0** [reply]
That is correct.

I am currently using yarn version 1.22.22

**Reply by snowystinger** [reply]
Could you clear your yarn cache? Maybe that's having some interference
